packages/cozy-client/src/queries/dsl.js

```diff
@@ -274,6 +275,35 @@ export const uploadFile = (file, dirPath) => ({
   dirPath
 })
 
+/**
+ * Creates a sort function from a definition.
+ *
+ * Used to sort query results inside the store when creating a file or
+ * receiving updates.
+ */
+
+export const makeSorterFromDefinition = definition => {
```

Comment:
can we have a docstring for the definition ?

Reply:
The docstring is just above, but was separated from the function with an empty line, I removed it.